modules/aws/vpc/README.md

```diff
@@ -0,0 +1 @@
+FIXME
```

Comment:
FIXME

Reply:
> FIXME

Oops, I hadn't meant to commit that :p.  Fixed with 2c59106 -> b23621f.